PopulateTools/gobierto
# Gobierto Data / Transform visualizations into images

Closes https://github.com/PopulateTools/issues/issues/1421


## :v: What does this PR do?

- Transform the vizzs into an image only when saved them.
- Store it the image into the API.
- Use the image to show a preview of the vizzs in the listings.
- Remove and reduce code 


## :mag: How should this be manually tested?

- Select a viz from [here](https://getafe.gobify.net/datos/licitaciones/visualizaciones) or another dataset.
- Edit and Save the vizz
- Then go to the [Visualizations Pages](https://getafe.gobify.net/datos/v/visualizaciones), [summary tab](https://getafe.gobify.net/datos/licitaciones/resumen), and [visualizations tab](https://getafe.gobify.net/datos/licitaciones/visualizaciones) to check if the image is shown.

You can also create a viz from the editor and check that the viz image is generated

## :eyes: Screenshots

### After this PR

https://user-images.githubusercontent.com/2649175/144232817-ac4d67e1-fb1e-4d7a-a326-b614e139d6ff.mov
